packages/cli/lib/services/response-saver.service.ts

```diff
@@ -1,38 +1,35 @@
 import fs from 'node:fs';
 import path from 'node:path';
-import type { AxiosResponse } from 'axios';
+import crypto from 'node:crypto';
+import type { AxiosRequestConfig, AxiosResponse } from 'axios';
 import type { Connection } from '@nangohq/shared';
 import type { Metadata } from '@nangohq/types';
 
+const FILTER_HEADERS = ['authorization', 'user-agent', 'nango-proxy-user-agent', 'accept-encoding', 'retries', 'host'];
```

Comment:
Any reason to not filter out `connection-id` and `provider-config-key`?

Reply:
Not at all. Fixed in 946e9dae